composer.json

```diff
@@ -130,7 +131,6 @@
         "symfony/doctrine-bridge": "<3.4",
         "symfony/security-core": "4.2.* <4.2.3",
         "symfony/swiftmailer-bundle": "2.6.* <2.6.2",
-        "symfony/twig-bundle": "4.1.0",
```

Comment:
Why did you remove this?

Reply:
because we're requiring `^4.2 || ^4.3` 😉